<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-4K1TBNQ" ignored="0" name="bibibuy" tests="4" failures="3" timestamp="2020-09-07T19:00:26 CST" time="1.038" errors="0">
  <testcase name="apiTest" time="0.743" classname="com.loex.tests.WebApiTests">
    <failure type="java.lang.AssertionError" message="接口请求失败.......请求URL:/order/create序号:2接口名称:币币交易用例标题:正确币币买入请求方法post请求参数{&amp;quot;type&amp;quot;: &amp;quot;1&amp;quot;, &amp;quot;side&amp;quot;: &amp;quot;BUY&amp;quot;, &amp;quot;price&amp;quot;: &amp;quot;1&amp;quot;, &amp;quot;volume&amp;quot;: &amp;quot;2&amp;quot;, &amp;quot;symbol&amp;quot;: &amp;quot;defilcny&amp;quot;}预期结果{&amp;quot;code&amp;quot;:&amp;quot;0&amp;quot;,&amp;quot;msg&amp;quot;:&amp;quot;成功&amp;quot;,&amp;quot;data&amp;quot;:null}实际结果{&amp;quot;code&amp;quot;:&amp;quot;10002&amp;quot;,&amp;quot;msg&amp;quot;:&amp;quot;用户未登录&amp;quot;,&amp;quot;data&amp;quot;:null} expected [{&amp;quot;code&amp;quot;:&amp;quot;0&amp;quot;,&amp;quot;msg&amp;quot;:&amp;quot;成功&amp;quot;,&amp;quot;data&amp;quot;:null}] but found [{&amp;quot;code&amp;quot;:&amp;quot;10002&amp;quot;,&amp;quot;msg&amp;quot;:&amp;quot;用户未登录&amp;quot;,&amp;quot;data&amp;quot;:null}]">
      <![CDATA[java.lang.AssertionError: 接口请求失败.......请求URL:/order/create序号:2接口名称:币币交易用例标题:正确币币买入请求方法post请求参数{"type": "1", "side": "BUY", "price": "1", "volume": "2", "symbol": "defilcny"}预期结果{"code":"0","msg":"成功","data":null}实际结果{"code":"10002","msg":"用户未登录","data":null} expected [{"code":"0","msg":"成功","data":null}] but found [{"code":"10002","msg":"用户未登录","data":null}]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at com.loex.tests.WebApiTests.apiTest(WebApiTests.java:117)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:110)
]]>
    </failure>
  </testcase> <!-- apiTest -->
  <testcase name="apiTest" time="0.031" classname="com.loex.tests.WebApiTests">
    <failure type="java.lang.AssertionError" message="接口请求失败.......请求URL:/order/create序号:3接口名称:币币交易用例标题:币对为空买入请求方法post请求参数{&amp;quot;type&amp;quot;: &amp;quot;1&amp;quot;, &amp;quot;side&amp;quot;: &amp;quot;BUY&amp;quot;, &amp;quot;price&amp;quot;: &amp;quot;1&amp;quot;, &amp;quot;volume&amp;quot;: &amp;quot;2&amp;quot;, &amp;quot;symbol&amp;quot;: &amp;quot;&amp;quot;}预期结果{&amp;quot;code&amp;quot;:&amp;quot;0&amp;quot;,&amp;quot;msg&amp;quot;:&amp;quot;成功&amp;quot;,&amp;quot;data&amp;quot;:null}实际结果{&amp;quot;code&amp;quot;:&amp;quot;10002&amp;quot;,&amp;quot;msg&amp;quot;:&amp;quot;用户未登录&amp;quot;,&amp;quot;data&amp;quot;:null} expected [{&amp;quot;code&amp;quot;:&amp;quot;0&amp;quot;,&amp;quot;msg&amp;quot;:&amp;quot;成功&amp;quot;,&amp;quot;data&amp;quot;:null}] but found [{&amp;quot;code&amp;quot;:&amp;quot;10002&amp;quot;,&amp;quot;msg&amp;quot;:&amp;quot;用户未登录&amp;quot;,&amp;quot;data&amp;quot;:null}]">
      <![CDATA[java.lang.AssertionError: 接口请求失败.......请求URL:/order/create序号:3接口名称:币币交易用例标题:币对为空买入请求方法post请求参数{"type": "1", "side": "BUY", "price": "1", "volume": "2", "symbol": ""}预期结果{"code":"0","msg":"成功","data":null}实际结果{"code":"10002","msg":"用户未登录","data":null} expected [{"code":"0","msg":"成功","data":null}] but found [{"code":"10002","msg":"用户未登录","data":null}]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at com.loex.tests.WebApiTests.apiTest(WebApiTests.java:117)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:110)
]]>
    </failure>
  </testcase> <!-- apiTest -->
  <testcase name="apiTest" time="0.066" classname="com.loex.tests.WebApiTests">
    <failure type="java.lang.AssertionError" message="接口请求失败.......请求URL:/order/create序号:4接口名称:币币交易用例标题:数量为空买入请求方法post请求参数{&amp;quot;type&amp;quot;: &amp;quot;1&amp;quot;, &amp;quot;side&amp;quot;: &amp;quot;BUY&amp;quot;, &amp;quot;price&amp;quot;: &amp;quot;1&amp;quot;, &amp;quot;volume&amp;quot;: &amp;quot;&amp;quot;, &amp;quot;symbol&amp;quot;: &amp;quot;defilcny&amp;quot;}预期结果{&amp;quot;code&amp;quot;:&amp;quot;0&amp;quot;,&amp;quot;msg&amp;quot;:&amp;quot;成功&amp;quot;,&amp;quot;data&amp;quot;:null}实际结果{&amp;quot;code&amp;quot;:&amp;quot;10002&amp;quot;,&amp;quot;msg&amp;quot;:&amp;quot;用户未登录&amp;quot;,&amp;quot;data&amp;quot;:null} expected [{&amp;quot;code&amp;quot;:&amp;quot;0&amp;quot;,&amp;quot;msg&amp;quot;:&amp;quot;成功&amp;quot;,&amp;quot;data&amp;quot;:null}] but found [{&amp;quot;code&amp;quot;:&amp;quot;10002&amp;quot;,&amp;quot;msg&amp;quot;:&amp;quot;用户未登录&amp;quot;,&amp;quot;data&amp;quot;:null}]">
      <![CDATA[java.lang.AssertionError: 接口请求失败.......请求URL:/order/create序号:4接口名称:币币交易用例标题:数量为空买入请求方法post请求参数{"type": "1", "side": "BUY", "price": "1", "volume": "", "symbol": "defilcny"}预期结果{"code":"0","msg":"成功","data":null}实际结果{"code":"10002","msg":"用户未登录","data":null} expected [{"code":"0","msg":"成功","data":null}] but found [{"code":"10002","msg":"用户未登录","data":null}]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at com.loex.tests.WebApiTests.apiTest(WebApiTests.java:117)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:110)
]]>
    </failure>
  </testcase> <!-- apiTest -->
  <testcase name="apiTest" time="0.054" classname="com.loex.tests.WebApiTests"/>
</testsuite> <!-- bibibuy -->
